R/write_eml.R

```diff
@@ -76,19 +76,20 @@ write_eml <- function(x, directory, derived_paragraph = TRUE) {
   eml$dataset$title <- x$title
 
   # Set abstract, with optional extra paragraph
-  para <- x$description
+  para <- x$description %>%
+    # Add <p></p> tags to each paragraph
```

Comment:
You are not adding <p></p> to each paragraph here, but rather wrapping the entire description as a single paragraph.

`x$description` can contain empty lines, so I suggest you use what you used in bird-tracking here:

```R
paragraphs <- unlist(strsplit(description_full, "<p>|</p>|\n"))
paragraphs <- paragraphs[paragraphs != ""]
```